includes/gateways/paypal/checkout-actions.php

```diff
@@ -154,20 +155,87 @@ function create_order( $purchase_data ) {
 			);
 		}
 
+		$order_subtotal = $purchase_data['subtotal'];
+		// Create an array of items for the order.
+		$items = array();
+		$i     = 0;
+		if ( is_array( $purchase_data['cart_details'] ) && ! empty( $purchase_data['cart_details'] ) ) {
+			foreach ( $purchase_data['cart_details'] as $item ) {
+				$item_amount = round( ( $item['subtotal'] / $item['quantity'] ) - ( $item['discount'] / $item['quantity'] ), 2 );
```

Comment:
I'm inclined to say we remove this `round()`. On line 174 it gets run through `edd_sanitize_amount()`, which already has logic for rounding and with a filter in place:

```php
/**
 * Filter number of decimals to use for prices
 *
 * @since unknown
 *
 * @param int $number Number of decimals
 * @param int|string $amount Price
 */
$decimals = apply_filters( 'edd_sanitize_amount_decimals', 2, $amount );
$amount   = number_format( (double) $amount, $decimals, '.', '' );
```